setup.py

```diff
@@ -36,15 +36,18 @@
 requirements = [
     'Flask>=0.10.1',
     'six>=1.7.2',
-    'invenio-records>=0.2.0'
-    # FIXME 'Invenio>=2.0.3',
+    'invenio-records>=0.2.0',
+    'sickle>=0.4',  # FIXME grab next release for full arXiv.org support
+    'Invenio>=2.0.3',
```

Comment:
Sure. However to make travis pass shall I add it to test_requirements or to `travis.yml`?